cmds/core/init/init.go

```diff
@@ -75,62 +74,14 @@ func main() {
 		}
 	}
 
+	libinit.SetEnv()
 	// Create the root file systems.
-	util.Rootfs()
+	libinit.CreateRootfs()
+	libinit.NetInit()
 
 	envs = os.Environ()
 	debug("envs %v", envs)
 
-	// install /env.
```

Comment:
where did some of this stuff go? It seems to be gone and some of it was needed.

Reply:
what is it for? can you explain? it includes calls to `sudo` (we don't even have a sudo) and stuff that doesn't tie into anything here at all

Reply:
ah ha, now it all comes clear with your comment! it was for when we integrated u-root into other initramfs like tinycore ... it's very good that this is gone :-)